source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -89,6 +89,25 @@ def label(self):
 
 class WordDocumentTextInfo(UIATextInfo):
 
+	def _get_locationText(self):
+		rects=self._rangeObj.getBoundingRectangles()

```

Comment:
Have you actually considered abstracting this into a method that will try to get a WordDocumentTextInfo based on the location of the UIA info? Could be beneficial for other features that are supported in the object model while not yet available in UIA. I guess it is ok to leave this pr as is now, though.